nucleus/core/kernel/src/main/manpages/fish/payara/enterprise/admin/commands/create-jvm-option.1

```diff
@@ -0,0 +1,201 @@
+           server
+               Specifies the DAS (default).
+
+           instance-name
+               Specifies a GlassFish Server instance.
+
+           cluster-name
+               Specifies a cluster.
+
+           configuration-name
+               Specifies a named configuration.
+
+       --profiler
+           Indicates whether the Java application launcher option is for the
+           profiler. The profiler must exist for this option to be true.
+           Default is false.
+
+OPERANDS
+       assignment
+           A single option assignment. The format of an option depends
+           on the following:
+
+           *   If the option has a name and a value, the format is
+               option-name=value.
+
+           *   If the option has only a name, the format is option-name. For
+               example, -Xmx2048m.
+
+           *   If the first option name could be misinterpreted as one or more
+               asadmin short options, the format is -- option-name. For
+               example, -server in the following command could be
+               misinterpreted as -se, the asadmin short forms for --secure and
+               --echo:
+
+                   create-jvm-option -server
+               To create the JVM option -server, instead use the command:
+
+                   create-jvm-option -- -server
+
+               Note
+               +-----------------------------------------+
+               | If an option name or option value       |
+               | contains a colon, the backslash (\)     |
+               | must be used to escape the colon in the |
+               | name or value. Other characters might   |
+               | also require an escape character. For   |
+               | more information about escape           |
+               | characters in subcommand options, see   |
+               | the asadmin(1M) man page.               |
+               +-----------------------------------------+
+
+EXAMPLES
+       Example 1, Setting system property
+
+               asadmin> create-jvm-option -Dunixlocation=/root/example
+               Successfully added JVM option: -Dunixlocation=/root/example
+               Command create-jvm-option executed successfully.
+
+       Example 2, Setting a Startup Parameter for the Java Application
+       Launcher
+           This example sets the maximum available heap size to 1024.
+
+               asadmin> create-jvm-option -Xmx1024m
+               Successfully added JVM option: -Xmx1024m
+               The previously configured maximum heap size (-Xmx512m) will be replaced by the new configuration: -Xmx1024m
+               Command create-jvm-option executed successfully.
+
+       Example 3, Setting a JVM Startup Parameter for the Profiler
+           This example sets a JVM startup parameter for the profiler.
+
+               asadmin> create-jvm-option --profiler=true -XX\:MaxPermSize=192m
+               Successfully added JVM option: -XX\:MaxPermSize=192m
```

Comment:
The colons don't need escaping, since we're not splitting on them with this command
```suggestion
               asadmin> create-jvm-option --profiler=true -XX:MaxPermSize=192m
               Successfully added JVM option: -XX:MaxPermSize=192m
```